pkg/interfaces/contracts/standalone-utils/IBalancerFeeBurner.sol

```diff
@@ -3,10 +3,31 @@
 pragma solidity ^0.8.24;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { WrappingDirection } from "../vault/VaultTypes.sol";
+
 import { SwapPathStep } from "../vault/BatchRouterTypes.sol";
 import { IProtocolFeeBurner } from "./IProtocolFeeBurner.sol";
 
 interface IBalancerFeeBurner is IProtocolFeeBurner {
+    struct StepsInfo {
+        uint256 currentIndex;
+        mapping(uint256 => SwapPathStep[]) steps;
+    }
```

Comment:
I'm not sure I fully follow why we need this.
Is it to avoid deleting existing paths? From what I can see we're always using the latest index.

If that's the case I think I prefer the previous approach. The `delete` operator should clean up the existing array completely, which is fine.